spec/support/features/session_helper.rb

```diff
@@ -286,7 +290,7 @@ def fill_in_code_with_last_phone_otp
 
     def fill_in_code_with_last_totp(user)
       accept_rules_of_use_and_continue_if_displayed
-      fill_in I18n.t('components.one_time_code_input.label'), with: last_totp(user)
+      fill_in 'code', with: last_totp(user)
```

Comment:
Same note, the previous code ensured the tests interact in a way similar to what we'd expect a user to do (a user doesn't know to find an input by its markup ID), but unfortunate if this is a technical limitation of Capybara.

Not proposing to do it here, but I'd like to see if this is something that Capybara could support (fixed in newer version, or configuration options, or upstream patch) or something we could build additional support for through our own custom helpers (similar to what we do with [accessibility matchers](https://github.com/18F/identity-idp/blob/main/spec/support/matchers/accessibility.rb)).

Reply:
> if this is something that Capybara could support (fixed in newer version, or configuration options, or upstream patch)

Related issue: https://github.com/teamcapybara/capybara/issues/2429

Reply:
There's some difficulty matching the input for code so I've opted for the more reliable method of matching by labelledby.